src/api/rpc_api/events.rs

```diff
@@ -159,6 +160,7 @@ where
 	Runtime: FrameSystemConfig + GetRuntimeBlockType,
 	Runtime::RuntimeBlock: BlockTrait + DeserializeOwned,
 	Runtime::Hashing: HashTrait<Output = Runtime::Hash>,
+	Runtime::Header: DeserializeOwned,
```

Comment:
Yes, at least at the moment because it calls `GetBlock::get_block(..)`.
I could create another `impl` with different trait bounds just for the new `get_genesis_block`, but if we merge the two traits then we will the bound on `Runtime::Header`